src/services/api.tsx

```diff
@@ -39,7 +44,109 @@ export const getPatients = async () => {
         if (name[0].given) patient.firstName = name[0].given.join(", ");
         if (name[0].family) patient.lastName = name[0].family;
       }
+      if (identifier) {
+        patient.identifier = identifier
+          .map((e: any) => {
+            return e.value;
+          })
+          .join(", ");
+      }
+
       return patient;
     }
   );
 };
+
+export const getPatientData = (patientId: string) => {
+  /*
+    getPatientData requests for data from Patient resourcce of id patientId
+    return a Patient object.
+  */
+  const getPatientData = async () => {
+    let response = await client.search({
+      type: "Patient",
+      patient: patientId,
+      query: {}
+    });
+    if (!response.data.entry) return; //patient not found
+    const patientData = response.data.entry[0];
+
+    const patient: Patient = {
+      id: patientData.resource.id
+    };
+
+    // Completing patient information with available data
+    if (patientData.resource.identifier) {
+      patient.identifier = patientData.resource.identifier[0];
```

Comment:
this is OK for now but later we will want to display all kinds of identifiers (not only the first one)
you can leave it like that for now

Reply:
oh right I think you joined identifiers with commas 20 lines below, right ?

Reply:
Wow sorry, the first patient.identifier completion has been deleted.